src/App.css

```diff
@@ -339,3 +339,101 @@ a.active {
 .location__title {
   text-align: left;
 }
+
+/*--------------------------------------------------------------------------*\
+   DIALOG
+\*--------------------------------------------------------------------------*/
+
+[data-reach-dialog-overlay] {
```

Comment:
TIL: It's new to me! I'm impressed how you could add styles with a `data-something` selector.